edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Diff.cs

```diff
@@ -8,56 +8,45 @@ namespace Microsoft.Azure.Devices.Edge.Agent.Core
 
     public class Diff
     {
-        public Diff(IList<IModule> updated, IList<string> removed)
+        public Diff(IList<IModule> added, IList<IModule> updated, IList<IModule> desiredStatusUpdated, IList<string> removed)
```

Comment:
We should add a comment that discussed what each of these `IList<IModule>`s is used for.

Reply:
Sure, will do.

Reply:
Added comments on the properties..